### DISCUSSION TITLE
Перенаправление трафика в контейнер

### DISCUSSION THREAD

**Discussion by skr7andy** [hufrea/byedpi#73] [General]
Здравствуйте. byeDPI запущен в контейнере в локальной сети с параметрами `-p 443 --split 1 --disorder 3+s --mod-http=h,d --auto --fake -1 --tlsrec 3+h` .
Если прописать в клиенте адрес и порт прокси, всё работает отлично, ютуб летает.
Но вопрос в другом, могу ли я при такой реализации на роутере (mikrotik) заворачивать трафик в этот контейнер, чтобы на каждом клиенте не настраивать прокси. В данный момент по доменам ютуба и dst-port 443 маркирую трафик, и маркированный трафик направляю маршрутизацией с GW = Docker-byeDPI, естесственно ничего не выходит ))
![image](https://github.com/user-attachments/assets/1038a39c-9758-4bb7-9022-d5c5e0cdcc07)

**Comment by InvisibleRain** [comment]
Вот тут ребята чем-то подобным занимались https://forum.mikrotik.com/viewtopic.php?t=203410&sid=8f0f6594d6a173bc1ee7595ba4ad839b#p1080756

**Comment by muzzy124** [comment]
Я извратился путем поднятия еще одной машинки с redsocks, который заворачивает все в socks5 byedpi. На микроте нужным клиентам отдаю шлюз с адресом машины redsocks

**Comment by wiktorbgu** [comment]
https://hub.docker.com/r/wiktorbgu/byedpi-hev-socks5-tunnel
собрал всё в одном контейнере :)
